### ISSUE TITLE
Bootstrap a Vue.js front-end

### ISSUE THREAD

**Issue by jkglasbrenner** [usnistgov/dioptra#87]
## Definition of Done

- [x] Have an initial build of the Vue.js UI produces a Hello World page that says "Welcome to Dioptra" and runs a simple REST API query to demonstrate working functionality (keeping in mind CORS)
- [x] Have a container build file that packages up the Vue.js front-end using the same tooling as the other images

**Comment by hbooth** [comment]
Need to integrate with new templated deployment generator
blocked by #83

**Comment by jkglasbrenner** [comment]
@hbooth I made progress on this since we spoke earlier today, I worked out the proxy rules needed in "production" (NGINX) and for "development" (Vite and Flask development servers) to integrate the Flask backend with the Vue.js frontend and keep CORS happy in both environments. I've pushed the changes I've made to the frontend-skeleton branch.

Before we merge to the dev branch, the one other thing we need to test is that the app continues to work when it is accessed via a FQDN and also after enabling HTTPS in NGINX. I know in some other apps I've had to configure a base URL in order to use a FQDN. I believe I found the place to set this in the Flask app, so I have configured things so that we can set a base URL via the environment variable `DIOPTRA_BASE_URL` in case that's necessary.

We can discuss in more detail tomorrow.
